src/Raven.Server/Smuggler/Migration/Migrator.cs

```diff
@@ -242,7 +242,6 @@ public long StartMigratingSingleDatabase(DatabaseMigrationSettings databaseMigra
                     result.AddInfo(message);
 
                     using (cancelToken)
-                    using (_httpClient)
```

Comment:
Can you explain?

Reply:
https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

Reply:
I do not think this is the case, since this client is ONLY used here inside Migrator. It is not shared with other operations.